### PR TITLE
Add Population Dislocation comparison tables

### LINKED ISSUE #292: Compare Population Dislocation Tables

### PR DESCRIPTION
**Pull request by ylyangtw**
Note: In PopDislocationCompare.jsx, map will be implemented in issue https://github.com/IN-CORE/incore-playbook/issues/317
image

### CODE REVIEW

File: src/components/childComponents/table/DislocationResultsTableCompare.jsx [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+	tableColumnHeadGreen: {
+		...tableColumnHead,
+		backgroundColor: "#A1BE2E"
+	},
+	tableColumnHeadRed: {
+		...tableColumnHead,
+		backgroundColor: "#DB4F60"
+	},
+	tableDataCell: {
+		width: "100px",
```
Comment (longshuicy) [inline src/components/childComponents/table/DislocationResultsTableCompare.jsx]: remove fixed width

File: src/components/childComponents/table/DislocationResultsTableCompare.jsx [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+		textAlign: "center",
+		border: "solid 1px rgba(46,91,255,0.08)"
+	},
+	tableDataCellDislocated: {
+		width: "100px",
```
Comment (longshuicy) [inline src/components/childComponents/table/DislocationResultsTableCompare.jsx]: remove fixed width

**Comment by ylyangtw** [comment]
@longshuicy I've removed widths. Here is the updated snapshot:
image

File: src/utils/common.js [hunk 3]
```diff
@@ -139,18 +139,22 @@ export function getShortHTML(html) {
...
 		}
-		return +(Math.round(+arr[0] + "e" + sig + (+arr[1] + scale)) + "e-" + scale);
+		return +(`${Math.round(`${+arr[0]  }e${  sig  }${+arr[1] + scale}`)  }e-${  scale}`);
 	}
 }
 
+export function diff(num1, num2) {
+	return num1-num2;
+}
+
```
Comment (longshuicy) [inline src/utils/common.js:157]: I like this utility function!

**Review by longshuicy — APPROVED** [review]

**Pull request merged**